lib/evss/letters/configuration.rb

```diff
@@ -6,7 +6,7 @@ class Configuration < EVSS::Configuration
       self.read_timeout = Settings.evss.letters.timeout || 55
 
       def base_path
-        "#{Settings.evss.url}/wss-lettergenerator-services-web/rest/letters/v1"
+        "#{Settings.evss.letters.url}/wss-lettergenerator-services-web/rest/letters/v1"
```

Comment:
You might make this safer by defaulting to the regular `Settings.evss.url` in the event that `Settings.evss.letters.url` == nil.  I.e.

```ruby
def base_path
  base = Settings.evss.letters.url || Settings.evss.url
  "#{base}/wss-lettergenerator-services-web/rest/letters/v1"
end
```